packages/patternfly-react/src/components/SessionTimeout/SessionTimeout.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Timer from '../../common/Timer';
+
+import { Icon, MessageDialog } from '../../index';
+
+export const SessionTimeoutDialog = props => (
+  <MessageDialog
+    show={props.show}
+    icon={
+      <Icon
+        className="session-timeout-dialog"
+        type="pf"
+        name="warning-triangle-o"
+      />
+    }
+    onHide={props.logoutFnc}
+    primaryActionButtonContent={props.continueText}
+    primaryAction={props.continueFnc}
+    secondaryActionButtonContent={props.logoutText}
+    secondaryAction={props.logoutFnc}
+    primaryContent={props.children}
```

Comment:
Would be better to allow primary and secondary content in the SessionTimeoutDialog so that the dialog can be consistent in styling with other message dialogs.